src/storage/index.js

```diff
@@ -13,22 +13,30 @@ const s3 = new aws.S3({
   credentials: config.s3.credentials,
 });
 
-const getFileUpload = (fileTag, maxFileSize) => {
-  const fileUpload = multer({
-    storage: multerS3({
-      s3,
-      bucket: config.s3.bucket,
-      acl: 'public-read',
-      key: (req, file, next) => {
-        next(null, `portal/profiles/${req.user.uuid}${path.extname(file.originalname)}`);
-      },
-    }),
+const uploadToS3 = async (folder, file, uuid) => {
+  const params = {
+    ACL: 'public-read',
+    Body: file.buffer,
+    Bucket: config.s3.bucket,
+    Key: `portal/${folder}/${uuid}${path.extname(file.originalname)}`,
+  };
+  const coverUrl = await s3.upload(params).promise()
+    .then((data) => data.Location)
+    .catch((error) => {
+      log.warn(`Failed to upload profile picture to S3: ${error}`);
+    });
+
+  return coverUrl;
```

Comment:
return the promise and let the controller catch the error; otherwise, this is gonna catch the error, write to log, and the controller sends a successful response for a failed request